### ISSUE TITLE
Can't communicate with module periodically.

### ISSUE THREAD

**Issue by snollygolly** [tessel/ir-attx4#32]
Periodically, I get the following message when running my project.

```
Computer:tessel-ir-utility snollygolly$ npm run tessel

> tessel-ir-utility@1.0.0 tessel /Users/snollygolly/tessel-ir-utility
> t2 run index.js

INFO Looking for your Tessel...
INFO Connected to Charmander.
INFO Building project.
INFO Writing project to RAM on Charmander (1744.384 kB)...
INFO Deployed.
INFO Running index.js...
Tessel IR Utility is now listening on http://192.168.1.79:8000
Could not retrieve firmware version and module ID. Attempting update.
Uploading new firmware...
Update finished!
events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: Unable to communicate with module. Are you sure it's plugged in?
    at commsError (/tmp/remote-script/node_modules/attiny-common/lib/index.js:52:18)
    at afterUpdate (/tmp/remote-script/node_modules/attiny-common/lib/index.js:69:15)
    at /tmp/remote-script/node_modules/attiny-common/lib/index.js:253:11
    at gotCRC (/tmp/remote-script/node_modules/attiny-common/lib/index.js:275:11)
    at null.<anonymous> (/usr/lib/node/tessel-export.js:296:31)
    at emitNone (events.js:67:13)
    at Socket.emit (events.js:166:7)
    at emitReadable_ (_stream_readable.js:411:10)
    at emitReadable (_stream_readable.js:405:7)
    at readableAddChunk (_stream_readable.js:157:11)
    at Socket.Readable.push (_stream_readable.js:110:10)
    at Pipe.onread (net.js:523:20)
```

It's always plugged in when this error happens, and firmly seated as well.  Usually re-running the command resolves the issue, but sometimes a full power cycle is needed.

It seems like it happens more when I run my project, halt it (control + c), wait a while (while I work on coding), and then re-run it.  I've started not halting until I'm ready to run it again, and that seems to make it happen significantly less.  Yes, I know that's strange, and it may completely be in my head.

**Comment by rwaldron** [comment]
I really dislike that these attx4 modules always check the firmware... @johnnyman727 there must be a better way?

**Comment by johnnyman727** [comment]
@rwaldron I'm open to suggestions but I'm not aware of a better way.

The reason why we check each time is because the version of the JavaScript driver *must* stay in sync with the version of firmware on the `attiny`. If not, they won't know how to talk to each other. When the checking mechanism works, it just takes a fraction of a second and isn't an issue.

**Comment by johnnyman727** [comment]
This should also be fixed by https://github.com/tessel/attiny-common/pull/9/files.

**Comment by johnnyman727** [comment]
Fixed by #36.
